client/components/mma/switch/review/SwitchReview.tsx

```diff
@@ -82,23 +82,28 @@ const PaymentDetails = (props: { subscription: Subscription }) => {
 			)}
 			{subscription.payPalEmail && (
 				<>
-					<span css={truncateCss}>
+					<span css={truncateCss} data-qm-masking="blocklist">
```

Comment:
You probably don't need the attribute here as it's already on the overall container so anything inside should already be masked.

Reply:
Removed as suggetsed